<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>/dev/ttyUSB0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openPortButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getParButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Param</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setParButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Param</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Uart Rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>8</number>
    </property>
    <property name="maxCount">
     <number>10</number>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>3</number>
    </property>
    <property name="maxCount">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Air Rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>6</number>
    </property>
    <property name="maxCount">
     <number>8</number>
    </property>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="maxCount">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>Max</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hi-mid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lo-mid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Min</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IO Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FEC</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
    <property name="maxCount">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>OC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PP/PU</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>WOR Timing</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
    <property name="maxCount">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>OFF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ON</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
    <property name="maxCount">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>Transparent</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fixed</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>8</number>
    </property>
    <property name="maxCount">
     <number>10</number>
    </property>
    <item>
     <property name="text">
      <string>250</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>750</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1250</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1750</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
